crd/multitenancy/api/v1alpha1/multitenantpodnetworkconfig.go

```diff
@@ -61,6 +62,8 @@ type MultitenantPodNetworkConfigStatus struct {
 	MacAddress string `json:"macAddress,omitempty"`
 	// Gateway IP
 	GatewayIP string `json:"gatewayIP,omitempty"`
+	// NIC Type
```

Comment:
can we add a comment to describe why we need this field? also suggestion to start comment with "NICType" same as var name

Reply:
Added